src/libstd/sys/windows/stdio_uwp.rs

```diff
@@ -0,0 +1,90 @@
+#![unstable(issue = "0", feature = "windows_stdio")]
+
+use crate::io;
+use crate::sys::c;
+use crate::sys::handle::Handle;
+
+pub struct Stdin {
+    #[allow(dead_code)]
```

Comment:
Since this is only compiled on UWP I think this may not be necessary?

Reply:
Yeah I guess I copy pasted a tiny bit too much :)

Reply:
Fixed